app/adapters/application.js

```diff
@@ -138,6 +140,11 @@ export default Adapter.extend(PouchAdapterUtils, {
         break;
       }
     }
+
+    if (!query.options.size) {
```

Comment:
@donaldwasserman I just tested this and I get the following error:
`vendor.js:40820 Error while processing route: patients.search Cannot read property 'size' of undefined TypeError: Cannot read property 'size' of undefined`
I believe this is because query.options is not passed in.